packages/eslint-config-cypress-ts/index.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    extends: [
+        '@owowagency/eslint-config-ts',
+        'plugin:cypress/recommended',
+    ],
+    rules: {
+        'cypress/no-unnecessary-waiting': 'warn',
+    },
```

Comment:
@barisbora, @polinaowow  shouldn't this be the correct linting config?
```suggestion
    extends: [
        '@owowagency/eslint-config-cypress-basic',
        '@owowagency/eslint-config-ts',
    ],
```
Make sure to upgrade the dependencies accordingly.

Reply:
I agree good to extend on typescript and cypress-basic